### ISSUE TITLE
Decouple number selection from notifier

### ISSUE THREAD

**Issue by MaxNoe** [fact-project/shifthelper#153]
Currently the FACTTwilioNotifier decides wheither to call the primary or the Fallback shifter.
I think this should be decoupled because we have multiplte notifiers. Last night it happend, that the call went out to the fallback shifter but telegram messages went to the primary shifter.

**Comment by dneise** [comment]
Yes, still the fallback shifter does not recieve Telegram messages, so in case the fallback is called and in the mean time the shifter ack'ed the alert. The fallback goes to the shifthelper webinterface and sees: all green and wonders why he was called.
